routes/users.js

```diff
@@ -24,6 +24,13 @@ router.put("/self/intro", authenticate, userValidator.validateJoinData, users.ad
 router.get("/:id/skills", users.getUserSkills);
 router.get("/:id/badges", getUserBadges);
 router.patch("/", authenticate, authorizeRoles([SUPERUSER]), users.nonVerifiedDiscordUsers);
+router.patch(
+  "/:id/roles",
```

Comment:
`roles` is not a noun please check the API guidelines and rename the API
https://swagger.io/resources/articles/best-practices-in-api-design/

Reply:
Hi @RitikJaiswal75 can you check this message on dev-backend
<img width="668" alt="image" src="https://github.com/Real-Dev-Squad/website-backend/assets/107163260/deccf3f5-3885-43f7-b517-e7596eb97ca8">
https://github.com/Real-Dev-Squad/website-backend/issues/1143

Reply:
@shubhamsigdar1 Could we call this API as `/:id/temporary/data` please? I don't want too many clients adopting it and causing a confusion later. Ideally, we should just fix `/users` the way @RitikJaiswal75 suggested. cc @vinayak-trivedi